Fix the testsuite failure:
FAIL: libffi.call/err_bad_abi.c -O0 -W -Wall (test for excess errors)

This patch will be sent upstream.

--- libffi-3.0.9/testsuite/libffi.call/err_bad_abi.c.orig	2016-01-13 09:44:54.467493090 -0800
+++ libffi-3.0.9/testsuite/libffi.call/err_bad_abi.c	2016-01-13 09:45:29.667133485 -0800
@@ -17,11 +17,9 @@
 	ffi_cif cif;
         void *code;
 	ffi_closure *pcl = ffi_closure_alloc(sizeof(ffi_closure), &code);
-	void* args[1];
 	ffi_type* arg_types[1];
 
 	arg_types[0] = NULL;
-	args[0] = NULL;
 
 	CHECK(ffi_prep_cif(&cif, 255, 0, &ffi_type_void,
 		arg_types) == FFI_BAD_ABI);
